raft/log.go

```diff
@@ -38,17 +38,20 @@ type raftLog struct {
 	applied uint64
 
 	logger Logger
+
+	maxMsgSize uint64
 }
 
 // newLog returns log using the given storage. It recovers the log to the state
 // that it just commits and applies the latest snapshot.
-func newLog(storage Storage, logger Logger) *raftLog {
+func newLog(storage Storage, logger Logger, maxMsgSize uint64) *raftLog {
```

Comment:
maybe provide another function `newLogWithSize`, so we don't need to change origin newLog in many places?

Reply:
Done.